data/types/food/food.go

```diff
@@ -27,6 +28,7 @@ func Meals() []string {
 		MealLunch,
 		MealOther,
 		MealSnack,
+		MealHypoglycemiaTreatment,
```

Comment:
Alphabetical order, please. This needs to be between `MealDinner` and `MealLunch`.